src/pint/observatory/special_locations.py

```diff
@@ -19,6 +25,8 @@
 
 from . import Observatory
 
+__all__ = ["load_special_locations"]
```

Comment:
Should this include the classes? If they're meant for people to subclass... in an ideal world the doc generator would only make documentation for what's in `__all__` but I couldn't make ours do that.